docs/reference/commandline/pause.md

```diff
@@ -24,13 +24,17 @@ Options:
       --help   Print usage
 ```
 
-The `docker pause` command suspends all processes in a container. On Linux,
-this uses the cgroups freezer. Traditionally, when suspending a process the
-`SIGSTOP` signal is used, which is observable by the process being suspended.
+The `docker pause` command suspends all processes the specified container.
```

Comment:
oh, sorry, no, I think this can stay as it was originally (`in a container`).

(or if you think it _is_ good to mention it, it should be `suspends all processes in the specified containers.`